javascript/chat-sub.js

```diff
@@ -0,0 +1,4 @@
+channel.presence.subscribe('enter', member => {
+  ui.showMemberEntered(member);
```

Comment:
We can add logger statement here, instead of ui showmember, for people new to js, this will be out of context and will
be hard to understand, since they can assume it's part of ably js SDK

Reply:
I don't think I agree.  We are trying to show code that actually could work, and achieves the goal. 
The point of showing customers code AFAIC is:
* Show how simple Ably is, and only show the code that matters
* Show code that could actually work.

Replacing code with comments, doesn't achieve point 2